client/my-sites/exporter/guided-transfer-details.jsx

```diff
@@ -0,0 +1,54 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import CompactCard from 'components/card/compact';
+import Gridicon from 'components/gridicon';
+
+const GuidedTransferDetails = ( { translate } ) => {
```

Comment:
Yep that sounds good to me. I wanted to keep things separate for now to avoid conflicts with all the various PRs and merging going on, but it makes sense to join the two into a single component. Let's do that once this is in master and the redirect is enabled (#6015)